src/main/java/org/codinjutsu/tools/jenkins/model/BuildParameter.java

```diff
@@ -0,0 +1,27 @@
+package org.codinjutsu.tools.jenkins.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.experimental.Accessors;
+import org.jetbrains.annotations.NotNull;
+
+@Data
+@Accessors(chain = true)
+@AllArgsConstructor
+public class BuildParameter {
+    @NotNull
+    private String name;
+    @NotNull
+    private String value;
```

Comment:
`value` should be `@Nullable`because password field or file parameter have no value